Sources/Fluent/Memory/Memory+Filters.swift

```diff
@@ -148,6 +148,10 @@ extension Node {
             case .or:
                 return passesOne(filters)
             }
+        case .raw(command: let command, values: let values):
+            // TODO: Throw error, memory DB doesn't support raw?
```

Comment:
Yeah we should definitely throw an error. Also mongodb isn't going to support raw.